pkg/config/database.go

```diff
@@ -68,7 +76,6 @@ func (d *Database) Open(logger *logging.Logger) (*icingadb.DB, error) {
 		uri := &url.URL{
 			Scheme: "postgres",
 			User:   url.UserPassword(d.User, d.Password),
-			Host:   net.JoinHostPort(d.Host, strconv.FormatInt(int64(d.Port), 10)),
 			Path:   "/" + url.PathEscape(d.Database),
 		}
```

Comment:
Just after this block, `uri.Host` is used. (Also, what happens if you enable TLS and use UNIX sockets?)

Reply:
(Nobody does this.)